draft-ietf-scitt-architecture.md

```diff
@@ -165,29 +170,34 @@ Issuers can register their Signed Statements on one or more Transparency Service
 
 # Introduction
 
-This document describes the generic, interoperable, and scalable SCITT architecture.
-Its goal is to enhance auditability and accountability across supply chains.
-
-In supply chains, downstream Artifacts are built upon upstream Artifacts.
-The complexity of traceability and quality control for these supply chains increases with the number of Artifacts and parties contributing to them.
-There are many parties who publish information about Artifacts:
-For example, the original manufacturer may provide information about the state of the Artifact when it left the factory.
-The shipping company may add information about the transport environment of the Artifact.
-Compliance Auditors may provide information about their compliance assessment of the Artifact.
-Security companies may publish vulnerability information about an Artifact.
-The original manufacturer may subsequently provide additional information about the manufacturing process they discovered after the Artifact left the factory.
-Some of these parties may publish information about their analysis or use of an Artifact.
-
-SCITT provides a way for Relying Parties to obtain this information in a way that is "transparent", that is, parties cannot lie about the information that they publish without it being detected.
-SCITT achieves this by having producers publish information in a Transparency Service, where Relying Parties can check the information.
+This document defines an architecture, a base set of extensible message structures and associated flows to make signed data items transparent via verifiable data structures maintained by corresponding transparency services.
+The goal of the transparency enabled by the Supply Chain Integrity, Transparency, and Trust (SCITT) architecture is to enhance auditability and accountability for signed data items (statements) that are about supply chain commodities (artifacts).
+Registering signed statements with a transparency service is akin to a notarization procedure.
+Transparency services therefore act like notaries that each record (register) a single, linear and irrevocable history of statements that were made.
+How these statements are managed or stored---beside the specified registration procedure---is out-of-scope of this document.
+Earlier, similar approaches for specific classes of artifacts exist, such as Certificate Transparency {{-CT}}.
+The SCITT approach follows a more generic paradigm than previous approaches.
+In this SCITT core architecture, the signed statements registered are opaque to the transparency services that act as a notary.
+This "content-agnostic" approach allows SCITT transparency services to be either integrated in existing solutions or to be an initial part of new emerging systems.
```

Comment:
```suggestion
Similar to a notary that isn't specific to a document type, the SCITT Architecture "content-agnostic" approach allows SCITT transparency services to be integrated into existing solutions or an initial part of emerging systems.
```

Reply:
This likely varies across jurisdictions, but where I am from, notaries have a relatively well-defined set of documents they will deal with, and will not accept arbitrary inputs. I think the earlier sentence gave the right idea.

Reply:
will confirm "nofix" in about 2h